Lib/fontbakery/profiles/googlefonts.py

```diff
@@ -3791,6 +3792,83 @@ def com_google_fonts_check_name_family_and_style_max_length(ttFont):
     yield PASS, "All name entries are good."
 
 
+@check(
+    id='com.google.fonts/check/has_unacceptable_control_chars',
```

Comment:
I'd like to suggest a simpler check-id: **com.google.fonts/check/control_chars**. What do you think?

But is this is really a family-wide check (I am not convinced it should be, though), then it would include the `"family"` keyword like this: **com.google.fonts/check/family/control_chars**

Reply:
refactored to `control_chars`!